geonode_mapstore_client/client/js/utils/ResourceUtils.js

```diff
@@ -525,3 +526,66 @@ export const parseMetadata = ({ entry } = {}) => {
     });
     return metadata;
 };
+
+/**
+ * Parse map response object
+ * @param {Object} mapResponse api response object
+ * @param {Object} resource optional resource object
+ * @returns {Object} new mao config object
+ */
+export const parseMapConfig = (mapResponse, resource = {}, type = 'map') => {
+
+    if (type !== 'map') {
+        return {
+            thumbnail: mapResponse.thumbnail_url,
+            src: mapResponse.href,
+            title: mapResponse.title,
+            description: mapResponse.raw_abstract,
+            credits: mapResponse.attribution,
+            sourceId: mapResponse.sourceId || 'geonode',
+            ...((mapResponse.subtype || mapResponse.type) === 'image' &&
+                { alt: mapResponse.alternate, src: mapResponse.href, ...(resource?.imgHeight && {imgHeight: resource?.imgHeight, imgWidth: resource?.imgWidth})})
+        };
+    }
```

Comment:
the name of this function is parseMapConfig so this operation does not really rely on this function because is parsing the other type of resources.
Please let's keep naming consistent so create a different function for image and video resources